src/components/UserSearch/UserSearch.jsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import SearchIcon from '@mui/icons-material/Search';
+import InputAdornment from '@mui/material/InputAdornment';
+import IconButton from '@mui/material/IconButton';
+import { styled } from '@mui/material/styles';
+import styles from './UserSearch.module.css';
+
+// searchbox styling
+const SearchBox = styled(TextField)(() => ({
+  '& fieldset': {
+    borderRadius: '25px',
+    backgroundColor: '#FAFAFC',
+    zIndex: -1,
+  },
+
+  '& .MuiInputBase-input': {
+    width: '250px',
```

Comment:
TODO: need to change this to be responsive. we can change this later when we make the buddy list.